dbld/rules

```diff
@@ -103,7 +106,7 @@ image-%:
 login: login-$(DEFAULT_IMAGE)
 login-%: setup
 	container=`$(DOCKER) ps | grep syslog-ng-$* | head -1 | cut -d ' ' -f1`; \
-	$(DOCKER) exec -ti  $$container /bin/bash
+	$(DOCKER) exec -ti  $$container gosu $(shell whoami) /source/dbld/shell
```

Comment:
As I understand the login target was introduced for getting a root shell inside a container, but with this change it will modified/broke.

Reply:
No, login was created so you can enter the same container multiple times. If was only a side effect that it was doing it as root. But I can agree that a root shell would be useful. Maybe a separate root-login target?

I can't remember why we can't do sudo inside the container though, that would be more handy. I think it was @szemere or @furiel who said that it was not secure or something.